### ISSUE TITLE
MaxSize, MinSize

### ISSUE THREAD

**Issue by slad99** [EricZimmerman/KapeFiles#330]
** KAPE version # **
0.9.5.0

**Describe the bug**
MaxSize and MinSize error

![imagem](https://user-images.githubusercontent.com/3330435/102261099-1d136d00-3f09-11eb-862d-f180625f5de7.png)

**Comment by EricZimmerman** [comment]
try validating it using Kape.exe and see if it works. it looks like i am missing those 2 properties in gkape target definition. easy fix, but try validating on command line

**Comment by slad99** [comment]
Works perfectly in KAPE.exe.
Thanks

**Comment by EricZimmerman** [comment]
ok, easy fix. will be in place for 0.9.6.0

i need to add these properties on the gkape side:

```
public long? MaxSize { get; set; }
public long? MinSize { get; set; }
```

**Comment by EricZimmerman** [comment]
Fixed in 0.9.6.0 which should be out soon
